libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/gutenberg/GutenbergEditorFragment.java

```diff
@@ -774,6 +794,33 @@ public void onClick(DialogInterface dialog, int id) {
         dialog.show();
     }
 
+    public void showFeaturedImageConfirmationDialog(final int mediaId) {
+        GutenbergDialogFragment dialog = new GutenbergDialogFragment();
+        dialog.initialize(
+                TAG_REPLACE_FEATURED_DIALOG,
+                getString(R.string.replace_current_title),
+                getString(R.string.replace_current_description),
+                getString(R.string.replace_current_confirmation),
+                getString(R.string.replace_current_cancel),
+                mediaId
+        );
+
+        dialog.show(getChildFragmentManager(), TAG_REPLACE_FEATURED_DIALOG);
+    }
+
+    private void setFeaturedImage(int mediaId) {
+        mEditorFragmentListener.updateFeaturedImage(mediaId, false);
+        // Necessary to update image in Post Settings
+        setFeaturedImageId(mediaId);
+        sendToJSFeaturedImageId(mediaId);
```

Comment:
Hmm, do we need the explicit `sendToJSFeaturedImageId()` call here? Not sure if I'm missing some codepath but looks like the `updateFeaturedImage()` two lines up is including a call to send the data to the JS side? 🤔

Reply:
Aha, yes, I missed that. 🤦‍♀️ Thank you for catching!

Reply:
> I've removed it in the latest commit

May I offer a suggestion (preference?) for how to link and close "the loop"? I personally add messages like "Fixed with <this commit hash link>" ([example here](https://github.com/wordpress-mobile/WordPress-Android/pull/6196#discussion_r127898836)) to save everyone a couple of seconds from scrolling to find the commit in question. I actually do that in all threads that end up in code changes to help focus the PR review. Of course, the reviewer will probably have a look at the overall diff as well but often times it helps to quickly check the commit that addresses a particular feedback and move on, until another overall pass.

Reply:
Noted for the future! Thank you so much, I appreciate tips around etiquette like that and see how that small change would make life a lot easier for the review. 🙇‍♀️